serverless/cli_tools/installing-kn.adoc

```diff
@@ -11,7 +11,7 @@ The Knative (`kn`) CLI does not have its own login mechanism. To log in to the c
 ifdef::openshift-enterprise[]
 For more information on installing the OpenShift (`oc`) CLI for your operating system and logging in with `oc`, see the xref:../../cli_reference/openshift_cli/getting-started-cli.adoc#cli-getting-started[OpenShift CLI getting started] documentation.
 endif::[]
-// need to wait til CLI docs are added to OSD for this link to work
+// need to wait til CLI docs are added to OSD and ROSA for this link to work
```

Comment:
Need to check if this is the case and whether commands for ROSA procedures need to be different, looks like ROSA has its own CLI `rosa` maybe instead of `oc`? Please check with QE.

Reply:
@mgencur @mvinkler Thanks for verifications. Could you please respond to Ashleigh's remaining questions here too?